packages/protocol/test-sol/devchain/e2e/utils.sol

```diff
@@ -54,4 +47,7 @@ contract Devchain is UsingRegistry, TestConstants {
     // TODO: Consider asserting that all contracts we expect are available in the Devchain class
     // (see list in migrations_sol/constants.sol)
   }
+  function setUp() public virtual override {
```

Comment:
forge should be ok with no setUp functions, but no big deal if it's not

Reply:
its required for test to run successfully. I added a comment to make it more clear